apps/test/unit/code-studio/progressReduxTest.js

```diff
@@ -732,6 +737,7 @@ describe('progressReduxTest', () => {
             app: 'maze',
             usesLab2: false,
             isValidated: false,
+            canHaveFeedback: undefined,
```

Comment:
nit: Will this reasonable ever be undefined or should we just make the tests false?

Reply:
The response from the server will never be undefined, but also the exact value of the parameter is irrelevant to the test and several others were marked as undefined instead of trying to craft a high-fidelity object.